armory/data/adversarial/pandas_proxy.py

```diff
@@ -0,0 +1,27 @@
+"""
+CSV reading fill-in to not require pandas dependency.
+
+A common pattern in adversarial datasets is:
+    import pandas
+    pandas.read_csv(path, header=None).to_numpy().astype("float32")
+
+This file is meant to replace that as follows:
+    import pandas_proxy
+    pandas_proxy.read_csv_to_numpy_float32(path, header=None)
+"""
+
+import csv
+
+import numpy as np
+
+
+def read_csv_to_numpy_float32(path, header=None) -> "np.array":
+    if header is not None:
+        raise NotImplementedError("non-None header not supported")
+
+    rows = []
+    with open(path, newline="") as f:
```

Comment:
We may want to add support for csv files to have headers...

Reply:
There are multiple places this could fail, not just headers. I did not want to implement the full CSV reading capability of pandas - just what I needed to replace these, since we only use them for building 4 specific datasets and all in the same way.